dev-client/src/screens/LocationScreens/components/soilId/messageBoxes/OfflineMessageBox.tsx

```diff
@@ -20,13 +20,17 @@ import {useTranslation} from 'react-i18next';
 import {AlertMessageBox} from 'terraso-mobile-client/components/messages/AlertMessageBox';
 import {Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
 
-export const OfflineMessageBox = () => {
+type OfflineMessageBoxProps = {
+  message: string;
+};
+
+export const OfflineMessageBox = ({message}: OfflineMessageBoxProps) => {
   const {t} = useTranslation();
 
   return (
-    <AlertMessageBox title={t('site.soil_id.matches.offline_title')}>
+    <AlertMessageBox title={t('general.offline_title')}>
```

Comment:
Issue: `general.offline_title` does not appear to be in `en.json` -- does this work?

Reply:
Ah, I see it's in PR #2573 -- I assume you'll figure out the right order to merge things for the strings to work out :)